make/prepare

```diff
@@ -264,6 +264,8 @@ else:
 
 storage_provider_name = rcp.get("configuration", "registry_storage_provider_name").strip()
 storage_provider_config = rcp.get("configuration", "registry_storage_provider_config").strip()
+# yaml requires 1 or more spaces between the key and value
+storage_provider_config = storage_provider_config.replace(":", ": ")
```

Comment:
What if you already specify a space after the ":" then we will have two spaces, will this still work?

Reply:
yes, the comment says "yaml requires 1 or more spaces between the key and value"